front/app/modules/commercial/smart_groups/components/UserFilterConditions/messages.ts

```diff
@@ -202,13 +202,13 @@ export default defineMessages({
     id: 'app.components.admin.UserFilterConditions.predicate_not_reacted_comment_in',
     defaultMessage: "didn't react to comment",
   },
-  predicate_budgeted_in: {
-    id: 'app.components.admin.UserFilterConditions.predicate_budgeted_in',
-    defaultMessage: 'spent budget',
+  predicate_voted_in: {
+    id: 'app.components.admin.UserFilterConditions.predicate_voted_in2',
+    defaultMessage: 'voted',
   },
-  predicate_not_budgeted_in: {
-    id: 'app.components.admin.UserFilterConditions.predicate_not_budgeted_in',
-    defaultMessage: "didn't spend budget",
+  predicate_not_voted_in: {
+    id: 'app.components.admin.UserFilterConditions.predicate_not_voted_in2',
+    defaultMessage: "didn't vote",
```

Comment:
The only thought I have is if admins will want more granular controls - I.e. if users have specifically submitted a budget versus a multiple vote or something. Since budgeting typically only happens once in a project, while there might be multiple voting phases :thinking: But that's a bit unrelated to this re-naming work though.

Reply:
Yes, think we decided to keep it as it is for now. I'll add it to the 'nice to haves' for a later ticket